<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wineman</string>
  </property>
  <property name="windowIcon">
   <iconset theme="wine"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="serverListTab">
      <attribute name="title">
       <string>Wine Servers</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableView" name="serverView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="serverButtonLayout">
         <item>
          <widget class="QPushButton" name="killServerButton">
           <property name="text">
            <string>Kill</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="taskManagerButton">
           <property name="text">
            <string>Task Manager</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="serverButtonVerticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="notificationsGroupBox">
         <property name="title">
          <string>Notifications</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="serverStartedNotificationCheckBox">
            <property name="text">
             <string>Show notification when a Wine server is started</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="serverStoppedNotificationCheckBox">
            <property name="text">
             <string>Show notification when a Wine server is stopped</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alwaysShowCheckBox">
         <property name="text">
          <string>Show tray icon when no servers are running</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="settingsTabVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
